pkg/flags/flags.go

```diff
@@ -33,6 +33,15 @@ func ConfigureAndParse(cmd *flag.FlagSet, args []string) {
 	maybePrintVersionAndExit(*printVersion)
 }
 
+// AddTraceFlags adds the traceCollector and ProbabilisticSamplingRate flags
```

Comment:
Nit: It looks like the flag names are `trace-collector` and `sampling-probability` as you mentioned in your PR description. If this comment is talking about the flags, it could be changed to use the flag names rather than the current names!